plugins/module_utils/network/sonic/config/bgp_af/bgp_af.py

```diff
@@ -512,8 +566,6 @@ def get_delete_route_advertise_request(self, vrf_name, conf_afi, conf_safi):
         url = '%s=%s/%s' % (self.network_instance_path, vrf_name, self.protocol_bgp_path)
         url += '/%s=%s/%s' % (self.afi_safi_path, afi_safi, self.l2vpn_evpn_route_advertise_path)
 
-        return ({'path': url, 'method': DELETE})
```

Comment:
Yes, I've added it back.

Reply:
I am marking this issue "resolved", but the fact that this bug didn't cause a sanity failure means that the only invocation of the function (for the "delete_all" case), was not being required to do the deletion because the related deletion(s) took place earlier in the regression test sequence.

Please modify the test sequence to leave a route advertise list in place for verification that it gets removed during "delete_all" handling.